src/ru/skypro/Main.java

```diff
@@ -1,8 +1,128 @@
 package ru.skypro;
 
-public class Main {
+import java.time.LocalDate;
+import java.util.Arrays;
+
+class Main {
+
+//Tesk1
+
+    public static void printLeapYear(int year) {
+        boolean leapYear = isLeapYear(year);
+        printLeapYearResult(year, leapYear);
+    }
+
+    private static boolean isLeapYear(int year) {
+        return year % 100 != 0 && year % 4 == 0 || year % 400 == 0;
+    }
+
+    private static void printLeapYearResult(int year, boolean leapYear) {
+        if (leapYear) {
+            System.out.println(year + " — високосный год");
+        } else {
+            System.out.println(year + " — невисокосный год");
+        }
+    }
+
+//Tesk2
+
+    public static int writeOS(int year, int OS) {
```

Comment:
переменные стоит именовать с маленьких букв
```suggestion
    public static int writeOS(int year, int os) {
```